var/spack/repos/builtin/packages/font-util/package.py

```diff
@@ -39,7 +39,6 @@ class FontUtil(AutotoolsPackage, XorgPackage):
         ['font-arabic-misc', '1.0.3', '3022b6b124f4cc6aade961f8d1306f67ff42e3b7922fb2244847f287344aefea'],
         ['font-bh-100dpi', '1.0.3', '817703372f080d6508cf109011b17f3572ff31047559fe82d93b487ca4e4e2d9'],
         ['font-bh-75dpi', '1.0.3', '720b6a513894bfc09a163951ec3dd8311201e08ee40e8891547b6c129ffb5fce'],
-        ['font-bh-ttf', '1.0.3', 'c583b4b968ffae6ea30d5b74041afeac83126682c490a9624b770d60d0e63d59'],
```

Comment:
What about something like this?
```
fonts = [f_r[0] for f_r in fonts_resource]
default_fonts = [f for f in fonts if f != 'font-bh-tff']
```
Then the values for the variant can be `fonts`, but the default is `','.join(default_fonts)`.

Then users can still request font-bh-tff if they want

We can also add `conflicts('font=font-bh-tff', when='platform=cray')`